ingestion/parsers/common/data_exporter.py

```diff
@@ -0,0 +1,63 @@
+import csv
+
+
+class DataExporter:
+  def __init__(self, data):
```

Comment:
your formatting doesn't conform to PEP-8, not so important for these scripts but better configure it accordingly